### DISCUSSION TITLE
Chat for maintainers

### DISCUSSION THREAD

**Discussion by PuneetGopinath** [Readme-Workflows/recent-activity#3] [General]
## 👋 Welcome all!

This is a general chat for maintainers.

**Comment by abhijoshi2k** [comment]
@PuneetGopinath Should we release this to marketplace?
It's all setup now

**Reply by PuneetGopinath** [reply]
Are you taking about 1.1.0 release, I think yes.

**Reply by PuneetGopinath** [reply]
I am thinking of a gh pages site, for this repo

**Reply by abhijoshi2k** [reply]
I feel the readme is enough. And once this is on marketplace, people will have access to this easily.

**Reply by PuneetGopinath** [reply]
Yes, also @Andre601 is requesting a new feature

**Reply by abhijoshi2k** [reply]
> Yes, also @Andre601 is requesting a new feature

I don't see any issues. Is it on previous repo?

**Reply by PuneetGopinath** [reply]
yes its on previous repo

**Reply by PuneetGopinath** [reply]
I think we should update the preview image in the readme

**Reply by abhijoshi2k** [reply]
> I think we should update the preview image in the readme

Can you do it please?

**Reply by abhijoshi2k** [reply]
I'll try to work on both issues tonight.

**Reply by PuneetGopinath** [reply]
Ok

**Reply by Andre601** [reply]
I just quickly drop in and say that I'll work on a quick PR for updating the Readme with some info and such... Just so that you guys have less work to do 👍

**Reply by PuneetGopinath** [reply]
Thanks

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Well when publishing next release do include a `v` in front of the version
This time you published 1.1.1, So the next time don't worry, I will use release-drafter to automate this, It doesn't look nice when you don't a `v` in front of the version

**Reply by abhijoshi2k** [reply]
Yeah sure. Actually I always add `v`. I forgot to do it for this one. I'll keep that in mind. Thanks

**Comment by PuneetGopinath** [comment]
@abhijoshi2k will I invite @Andre601 as collaborator

**Reply by abhijoshi2k** [reply]
Yes. Do it.

**Reply by abhijoshi2k** [reply]
FYI, I've started working on #5 
I'll complete it by tomorrow night probably!

**Reply by Andre601** [reply]
![image](https://user-images.githubusercontent.com/11576465/117546824-ba904a00-b02c-11eb-9915-f8452866834f.png)

That's an interesting GitHub Bug xD

**Reply by abhijoshi2k** [reply]
> ![image](https://user-images.githubusercontent.com/11576465/117546824-ba904a00-b02c-11eb-9915-f8452866834f.png)
> 
> That's an interesting GitHub Bug xD
Maybe you will get access after you are collaborator.

@PuneetGopinath did u send the invite?

**Reply by Andre601** [reply]
FYI I can see the Project, that's not the problem. This just poped up when I viewed a specific card (The one about #5) and the project then updated.

**Reply by abhijoshi2k** [reply]
Weird bug. I guess difficult to reproduce as well xD

**Reply by PuneetGopinath** [reply]
> @PuneetGopinath did u send the invite?

Now sent.

**Reply by PuneetGopinath** [reply]
Is the CODEOWNERS file ok?

**Reply by PuneetGopinath** [reply]
> Weird bug. I guess difficult to reproduce as well xD

I didn't send invitation in the night. I didn't see your msg because I was busy afterwards with some other work.

**Reply by PuneetGopinath** [reply]
You could have also checked whether I sent in the settings, and if not sent you could have sent one.

**Reply by abhijoshi2k** [reply]
> You could have also checked whether I sent in the settings, and if not sent you could have sent one.

Yeah I know. I was so much into figuring out how to resolve those issues. I couldn't think of other things xD

**Reply by abhijoshi2k** [reply]
> Is the CODEOWNERS file ok?

Just viewed it. It's fine!

**Reply by PuneetGopinath** [reply]
I think I should put @Andre601 to org as a owner

**Reply by abhijoshi2k** [reply]
I agree. His contributions are great. Let's do it.
Also I've released v1.2.2
@Andre601 You can make the commit.

**Reply by abhijoshi2k** [reply]
> I think I should put @Andre601 to org as a owner

Shall I send him the invite?

**Reply by PuneetGopinath** [reply]
Yep

**Comment by abhijoshi2k** [comment]
@PuneetGopinath  I closed the pr by mistake. Sorry for that. I have restored the branch. Can you add the changelog and release the new version?

**Reply by PuneetGopinath** [reply]
I forgot to check that I have already Updated the ChangeLog

**Reply by PuneetGopinath** [reply]
Can't we release after merging #14?

**Reply by abhijoshi2k** [reply]
> Can't we release after merging #14?

We'll release after the feature requested by @Andre601 is completed. Will open the pr in few hours.

**Reply by PuneetGopinath** [reply]
Yes, `URL_FORMAT` right?

**Reply by abhijoshi2k** [reply]
> Yes, `URL_FORMAT` right?

Yes.

**Comment by abhijoshi2k** [comment]
@PuneetGopinath @Andre601 Should we release new version?

**Reply by Andre601** [reply]
Sure. Why not

**Reply by abhijoshi2k** [reply]
@Andre601 Do you know why this activity isn't terminating?
https://github.com/Readme-Workflows/recent-activity/runs/2539370218

**Reply by Andre601** [reply]
> @Andre601 Do you know why this activity isn't terminating?
> https://github.com/Readme-Workflows/recent-activity/runs/2539370218

No idea.
It's a gh page build it seems

**Reply by abhijoshi2k** [reply]
> > @Andre601 Do you know why this activity isn't terminating?
> > https://github.com/Readme-Workflows/recent-activity/runs/2539370218
> 
> No idea.
> It's a gh page build it seems

But gh pages is disabled.

**Comment by Andre601** [comment]
Just updated my Workflow and the action for some reason used the PR_CLOSED text rather than the PR_MERGED one despite the PR being merged and not closed.

PR in question: https://github.com/purrbot-site/PurrBot/pull/348
README (I updated it which is why this is a commit branch):
https://github.com/Andre601/Andre601/blob/227aefd8d9ee59066f392bc6844930d5ebc411c5/README.md

**Reply by abhijoshi2k** [reply]
I just fixed the issue on optimization branch.
So after you review it, we will merge.

**Comment by Andre601** [comment]
Would it be alright if I take some minor changes to the release-drafter.yml file?

**Reply by abhijoshi2k** [reply]
Yes I think it'd be alright.
If you are planning to open PR within a day or two, open it on `optimization` branch.

**Reply by PuneetGopinath** [reply]
No need to ask for editing files, just submit a pr.

**Comment by abhijoshi2k** [comment]


**Reply by Andre601** [reply]
I say go for it

**Reply by Andre601** [reply]
If it's just publishing the release (which I assume it is) could I also do this real quick

**Reply by abhijoshi2k** [reply]
Do what exactly? @Andre601

**Reply by PuneetGopinath** [reply]
Just do it

**Reply by abhijoshi2k** [reply]
Released v1.2.1

**Comment by PuneetGopinath** [comment]
@abhijoshi2k Can you tell me how did you generate the photo in your medium article about this recent-activity repo?

I have seen that type of photos many places but I don't know how to generate one?

Preview of that image:
![image](https://miro.medium.com/max/2690/1*MlylV-rsw-yNVTdHnxWMDQ.png)

**Reply by abhijoshi2k** [reply]
Oh I just took a screenshot of my profile page, cropped it and used some online image editor.
Not much efforts xD

**Reply by PuneetGopinath** [reply]
> Oh I just took a screenshot of my profile page, cropped it and used some online image editor.
> Not much efforts xD

I know you took a screenshot, but what online editor?

**Reply by abhijoshi2k** [reply]
I remember now. It wasn't online editor. It was Microsoft Paint 3d xD

**Reply by PuneetGopinath** [reply]
> I remember now. It wasn't online editor. It was Microsoft Paint 3d xD

Ok. This photo truly attracts people, that's why I asked.

**Reply by abhijoshi2k** [reply]
It's a good took for basic editing. Although a bit annoying 🤣

**Reply by PuneetGopinath** [reply]
Annoying? Why?

**Reply by abhijoshi2k** [reply]
It's not exactly user friendly

**Comment by abhijoshi2k** [comment]
@PuneetGopinath can you please elaborate the issue you have opened?

**Reply by PuneetGopinath** [reply]
GitHub Marketplace could not find v1 in our releases, that's the error

**Reply by abhijoshi2k** [reply]
But why do we need that. I don't understand.
We will directly use:
`Readme-Workflows/recent-activity@v1.2.1`
or
`Readme-Workflows/recent-activity@main`
We will not need v1

**Comment by abhijoshi2k** [comment]
@Andre601 @PuneetGopinath I think v1.2.2 is ready to be released.
Is there any change pending?

**Reply by Andre601** [reply]
You should've waited for me to update the changelog completely before merging the pr. Now I have to make an extra commit to update the release date.

**Reply by abhijoshi2k** [reply]
I again apologize for the same.

**Reply by PuneetGopinath** [reply]
No changes, You can release

**Comment by Andre601** [comment]
Looks like we can now display/share videos on READMEs
https://github.blog/changelog/2021-05-13-video-uploads-now-generally-available/

That could be useful to f.e. have a small video showcasing how to set up the action.

**Reply by abhijoshi2k** [reply]
That's a great idea.
However, I won't be able to make the video as there's an airport near my house and loud plane noises every 2 min.
@PuneetGopinath or @Andre601 will you be able to do this?

**Reply by PuneetGopinath** [reply]
> Looks like we can now display/share videos on READMEs
> https://github.blog/changelog/2021-05-13-video-uploads-now-generally-available/
> 
> That could be useful to f.e. have a small video showcasing how to set up the action.

Yes, but screen recording I can't

**Reply by Andre601** [reply]
> That's a great idea.
> However, I won't be able to make the video as there's an airport near my house and loud plane noises every 2 min.
> @PuneetGopinath or @Andre601 will you be able to do this?

Not sure why this would affect the video. You don't need to record any audio, especially if you record your PC screen...

**Reply by abhijoshi2k** [reply]
Considering the complexity of parameters and placeholders, I feel video is better than gifs @PuneetGopinath

**Reply by abhijoshi2k** [reply]
> > That's a great idea.
> > However, I won't be able to make the video as there's an airport near my house and loud plane noises every 2 min.
> > @PuneetGopinath or @Andre601 will you be able to do this?
> 
> Not sure why this would affect the video. You don't need to record any audio, especially if you record your PC screen...

Wouldn't audio be required to explain the placeholders? especially `{URL}`

**Reply by Andre601** [reply]
I don't think we need too much info here.
I also doubt that we can make a too big video anyway since GitHub has an size limit for those files after all.

**Reply by PuneetGopinath** [reply]
Yep

**Reply by abhijoshi2k** [reply]
@PuneetGopinath has created a template repo. We can directly show how to pick up the template and add to our repo and maybe show few minor edits like disabling comments, adding a placeholder, etc.
How does that sound? @Andre601 
I'll do the screen recording. No issues if there's no audio.

**Reply by PuneetGopinath** [reply]
@abhijoshi2k I said that repo is not yet ready!!

**Reply by abhijoshi2k** [reply]
> @abhijoshi2k I said that repo is not yet ready!!

I am planning to open a PR soon

**Reply by abhijoshi2k** [reply]
We will do it only after it's ready.

**Reply by PuneetGopinath** [reply]
Ok, do it when it's ready

**Reply by abhijoshi2k** [reply]
Alright

**Comment by Andre601** [comment]
I currently consider making a discord server just so we have a different chat area because the amount of e-mails I sometimes get is huge xD

If you're interested can I make one real quick and share the invite here afterwards.

**Reply by PuneetGopinath** [reply]
Right, wait I can create it.

**Reply by PuneetGopinath** [reply]
Can I create it, reply fast.

**Reply by Andre601** [reply]
I could've made one too... ~~Especially since I have a lot of experience making and managing servers~~

Question now is, if it should stay a more or less private one, or if others are allowed to also join it.

**Reply by PuneetGopinath** [reply]
https://discord.gg/rh7ABZd7wv

**Reply by PuneetGopinath** [reply]
Continue chat there

**Reply by abhijoshi2k** [reply]
I'll be honest....I haven't used discord as admin much. But it sounds better than here. As I am facing same issue with so many emails xD.

**Reply by PuneetGopinath** [reply]
Yep

**Comment by PuneetGopinath** [comment]
Finally I finished the [Community profile](https://github.com/Readme-Workflows/recent-activity/community)

**Reply by abhijoshi2k** [reply]
Great. Everything's looking perfect! @PuneetGopinath

**Reply by PuneetGopinath** [reply]
One thing left, I will commit it now.

**Reply by PuneetGopinath** [reply]
Committed "Update README.md"

**Comment by Andre601** [comment]
Locked this as Discussion moved to Discord.
